dask_sql/physical/rel/custom/create_memory_table.py

```diff
@@ -38,7 +38,11 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
         # Rust create_memory_table instance handle
         create_memory_table = rel.create_memory_table()
 
-        schema_name, table_name = context.schema_name, create_memory_table.getName()
+        # TODO: can we avoid hardcoding the schema name?
+        schema_name, table_name = (
+            context.schema_name,
+            create_memory_table.getTableName(),
```

Comment:
I reviewed `LogicalPlan.schema(),` and it looks like the implementation for `CreateMemoryTable` is incorrect because it returns the schema of the input rather than the schema of the newly created table. I recommend filing a bug against DataFusion for this.

Reply:
Thanks for looking into this @andygrove! Opened https://github.com/apache/arrow-datafusion/issues/5004 to track this

Reply:
@andygrove I was looking into apache/arrow-datafusion#5004 but I don't quite understand the motivation here: the difference between the schema of the input and the schema of the newly create table is only the table name. Is the idea to make it override the qualifiers of `DFField`s from the input?

Reply:
cc: @charlesbluca